hudi-common/src/main/java/org/apache/hudi/metadata/FileSystemBackedTableMetadata.java

```diff
@@ -196,8 +196,8 @@ private List<String> getPartitionPathWithPathPrefixUsingFilterExpression(String
                     } else if (!path.getName().equals(HoodieTableMetaClient.METAFOLDER_NAME)) {
                       return Pair.of(Option.empty(), Option.of(path));
                     }
-                  } else if (path.getName()
```

Comment:
if we can add partitionMetadata for partition_stats folder, it will pass in Line 191. Not sure how to do that yet. So workaround here.